src/resources/js/tickets-attendees.js

```diff
@@ -32,8 +32,23 @@ var tribe_event_tickets_attendees = tribe_event_tickets_attendees || {};
 			var $pointer = $( AttendeesPointer.target ).pointer( options ).pointer( 'open' ).pointer( 'widget' );
 		}
 
-		$( 'input.print' ).on( 'click', function( e ) {
+		$( 'input.print' ).on( 'click', function() {
+			$( window ).trigger( 'tribe-tickets.attendees-report.before-print' );
```

Comment:
this is but a ticket related action that will apply to any ticket-able post type, does it make sense to use `events.`?